pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -43,6 +43,26 @@ func getDefaultInstrumentation(agentConfig *adapters.CwaConfig) (*v1alpha1.Instr
 		}
 	}
 
+	javaEnvVar := []corev1.EnvVar{
+		{Name: "OTEL_AWS_APP_SIGNALS_ENABLED", Value: "true"}, //TODO: remove in favor of new name once safe
+		{Name: "OTEL_AWS_APPLICATION_SIGNALS_ENABLED", Value: "true"},
+		{Name: "OTEL_TRACES_SAMPLER_ARG", Value: "endpoint=http://cloudwatch-agent.amazon-cloudwatch:2000"},
+		{Name: "OTEL_TRACES_SAMPLER", Value: "xray"},
+		{Name: "OTEL_EXPORTER_OTLP_PROTOCOL", Value: "http/protobuf"},
+		{Name: "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT", Value: exporterPrefix + "cloudwatch-agent.amazon-cloudwatch:4316/v1/traces"},
+		{Name: "OTEL_AWS_APP_SIGNALS_EXPORTER_ENDPOINT", Value: exporterPrefix + "cloudwatch-agent.amazon-cloudwatch:4316/v1/metrics"}, //TODO: remove in favor of new name once safe
+		{Name: "OTEL_AWS_APPLICATION_SIGNALS_EXPORTER_ENDPOINT", Value: exporterPrefix + "cloudwatch-agent.amazon-cloudwatch:4316/v1/metrics"},
+		{Name: "OTEL_METRICS_EXPORTER", Value: "none"},
```

Comment:
This cannot be "none" for the JMX case. Setting it to "none" will disable all metrics from being exported. We only need to set it to "none" if JMX isn't enabled for the pod.

Reply:
also this code path runs specifically when an Instrumentation CR is not found...if customers happen to have their own instrumentation CR for appsignals, these jmx env vars won't be injected. 

This means customers with a custom appsignals setup will have to manually add these vars to their Instrumentation spec. Is that expected?

Reply:
How do we currently handle that case for appsignals?

Reply:
I'm not understanding this. Mind if we huddle?